pkg/console/agent.go

```diff
@@ -0,0 +1,138 @@
+package console
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/pluralsh/plural/pkg/helm"
+	"helm.sh/helm/v3/pkg/action"
+
+	"github.com/gofrs/flock"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/helmpath"
+	"helm.sh/helm/v3/pkg/repo"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"sigs.k8s.io/yaml"
+)
+
+var settings = cli.New()
+
+const (
+	repoName  = "deployop"
+	repoUrl   = "https://pluralsh.github.io/deployment-operator"
+	namespace = "console"
```

Comment:
the namespace for this should actually be `plrl-deploy-operator` (it'll be installed in non-plural clusters so that seemed more general)

Reply:
we also are probably going to need to use the `create-namespace` flag as a result of that